src/main/java/seedu/address/model/person/Person.java

```diff
@@ -45,38 +46,80 @@ public Person(Name name, Map<String, Phone> numbers, Map<String, Email> emails,
         this.tags.addAll(tags);
     }
 
+    /**
+     * Returns an empty person with nothing values.
+     *
+     * @return an empty person with nothing values
+     */
+    public static Person getEmptyPerson() {
```

Comment:
Is getEmptyPerson() still required if we are passing the person straight to EditCommand?

Reply:
Right now, there are 17 instances of getEmptyPerson() being used, 1 in EditCommandParser and 16 in EditCommand and EditCommandParser Tests. Instead of having to have a long line of code to instantiate a new Person with all their empty fields, I think it is good to abstract them out as getEmptyPerson()